gascity/commands/adt-mcp-http/run.sh

```
@@ -0,0 +1,29 @@
+if [ -z "$RIG_PATH" ]; then
+  SEARCH_DIR="${GC_CITY_PATH:-$(CDPATH= cd -- "$(dirname "$0")/../.." && pwd)}"
+  while [ "$SEARCH_DIR" != "/" ]; do
+    if [ -f "${SEARCH_DIR}/packages/adt-mcp/dist/bin/adt-mcp-http.mjs" ]; then
+      RIG_PATH="$SEARCH_DIR"
+      break
+    fi
+    SEARCH_DIR="$(dirname "$SEARCH_DIR")"
+  done
+fi
+if [ -z "$RIG_PATH" ]; then
+  RIG_PATH="$(CDPATH= cd -- "$(dirname "$0")/../.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)